extension.js

```diff
@@ -40,49 +42,59 @@ function activate(context) {
       return lineText.trim().startsWith('require')
     }
 
+    /**
+     * @param {string[]} namespaceDefinitions
+     */
+    function trimNamespaceKeywords(namespaceDefinitions) {
+      return namespaceDefinitions.map((namespaceDefinition) => {
+        return (namespaceDefinition
+          .replace('class ', '')
+          .replace('module ', '')
+        )
```

Comment:
A verdade é que a gente não tava dando trim nas definições de classe/module. Isso fazia com que a indentação gerasse uns 'leading whitespaces', que pareciam ser só um por cauisa de aguma formatação estranha que o vscode faz: 

![image](https://user-images.githubusercontent.com/22344659/77241349-a8a4db00-6bcf-11ea-8a58-e01207085d92.png)

![image](https://user-images.githubusercontent.com/22344659/77241353-b8242400-6bcf-11ea-915f-b696d63ce7f6.png)


Solução: 

```suggestion
        .trim()
        )
```

seria legal dar uma olhada se esse comportamento é o normal do vscode.

Reply:
Deu certo!